src/IoTUtils.py

```diff
@@ -3,6 +3,7 @@
 from uuid import uuid1
 from boto3 import client
 from types_boto3_iot import IoTClient
+import botocore
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The code uses broad library imports instead of importing specific required class, which consumes unnecessary memory and makes code maintenance harder by obscuring actual library usage. To optimize performance and improve code clarity, use targeted imports with &#39;from library import specific_class&#39; syntax. Learn More https://docs.python.org/3/tutorial/modules.html.